src/components/pages/TeamPage.jsx

```diff
@@ -20,66 +21,23 @@ const TeamPage = () => {
 
         <h2 className="font-spartan font-semibold text-2xl lg:text-3xl"> Team Mentors</h2>
         <div className="w-full flex flex-wrap items-center justify-center mb-4">
-          <MentorCard BlobImage={GreenBlobImage} teamMemberData={teamMembers[0]} imgShift="12" textShift="8"/>
-          <MentorCard BlobImage={YellowBlobImage} teamMemberData={teamMembers[1]} cardShift="2"/>
-          <MentorCard BlobImage={PurpleBlobImage} teamMemberData={teamMembers[2]} cardShift="6"/>
+          <MentorCard BlobImage={GreenBlobImage} teamMemberData={teamMembers[0]} cardShift="0" imgShift="24" textShift="8" />
```

Comment:
A good rule of thumb is: whenever you are indexing an array multiple times, you can probably map it instead. Let's apply the same logic of mapping through our array for the mentor section.

Reply:
Since each mentor card has a different blob image (different shape and side), mentor cards require individual adjusting (padding and margins so that the mentor image and text stay inside blobs). I believe mentor cards cannot be processed with a map.

Reply:
You would have to build those properties into the mentor object and pass that onto the component and then using the index you could provide additional styles to only specific components.But that's a little more involved to just render four images. No need to apply for now but just keep in mind for the future.